src/pages/AppConfigure.vue

```diff
@@ -13,13 +13,20 @@ import {
   setAppName,
   setClientIds,
   setDefaultLimit,
+  enableUiMode,
 } from '@/services/smart-contract.service'
-import type { ConfigureTab, ConfigureTabType } from '@/utils/constants'
+import {
+  WalletMode,
+  type ConfigureTab,
+  type ConfigureTabType,
+} from '@/utils/constants'
```

Comment:
can we separate `types` and `constants` in different files?

Reply:
I'll do it in a seperate PR. ref: https://team-1624093970686.atlassian.net/browse/AR-2779